math/mathcore/inc/TStatistic.h

```diff
@@ -44,7 +44,7 @@ class TStatistic : public TObject {
 
 public:
 
-   TStatistic(const char *name = "") : fName(name), fN(0), fW(0.), fW2(0.), fM(0.), fM2(0.), fMin(TMath::Limits<Double_t>::Max()), fMax(TMath::Limits<Double_t>::Min()) { }
+   TStatistic(const char *name = "") : fName(name), fN(0), fW(0.), fW2(0.), fM(0.), fM2(0.), fMin(TMath::Limits<Double_t>::Max()), fMax(-TMath::Limits<Double_t>::Max()) { }
```

Comment:
is `-numeric_limits<double>::max()` guaranteed to be the same as `numeric_limits<double>::lowest()`? If not, we probably want the latter